crates/web-sys/webidls/Clipboard.webidl

```diff
@@ -1,43 +1,29 @@
 /* -*- Mode: IDL; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
 /*
  * Clipboard API and events
- * W3C Working Draft, 4 June 2021
+ * Editor’s Draft, 4 June 2024
+ *
  * The origin of this IDL file is:
- * https://www.w3.org/TR/2021/WD-clipboard-apis-20210604/
+ * https://w3c.github.io/clipboard-apis/#idl-index
  */
 
 partial interface Navigator {
   [SecureContext, SameObject] readonly attribute Clipboard? clipboard;
```

Comment:
I left this as optional `?`, but I am not sure if that is required. i.e. what happens in browser without `navigator.clipboard` ?

Reply:
It would panic, but this is problem we won't solve here, but would require a more general solution.

So please go ahead and remove it.